src/index.html

```diff
@@ -11,6 +11,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
```

Comment:
No need to separate parent and child element
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
```